src/main/java/seedu/duke/command/SearchCommand.java

```diff
@@ -0,0 +1,89 @@
+package seedu.duke.command;
+
+import seedu.duke.anime.Anime;
+import seedu.duke.anime.AnimeData;
+import seedu.duke.exception.AniException;
+import seedu.duke.human.User;
+import seedu.duke.storage.StorageManager;
+
+import java.util.Arrays;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import static seedu.duke.logger.AniLogger.getAniLogger;
+
+public class SearchCommand extends Command {
+    protected static final String ASSERT_SEARCH_TERM_EMPTY = "Empty Search String!";
+    protected static final String ID_HEADER = "[ID:";
+    protected static final String ID_CLOSER = "] ";
+    protected static final String NO_RESULTS_FOUND = "No results found!";
+    protected static final String SEARCHING_BY_GENRE = "Searching By Genre";
+    protected static final String SEARCHING_BY_ANIME_NAME = "Searching By Anime Name";
+    protected static final String SEARCH_TYPE_INVALID = "Something went wrong with search input";
+    protected static final String SEARCH_TYPE_INVALID_LOG = "Search Type has the wrong values.";
+    protected static final int SEARCH_BY_NAME = 0;
+    protected static final int SEARCH_BY_GENRE = 1;
+
+    private static final Logger LOGGER = getAniLogger(SearchCommand.class.getName());
+
+    protected String searchTerm;
```

Comment:
I think for command, private will be the better access modifiers, as there will not be any subclass to SearchCommand.

Reply:
I have made the changes, in all affected areas. Thank you for pointing it out.